### ISSUE TITLE
[FEATURE] Combine 'Docs Menu' and top navigation menu into one

### ISSUE THREAD

**Issue by tommyalatalo** [onweru/compose#82]
**I have checked all the prerequisites below and I don't see this is available**

- [X] Read the README.md
- [X] Have the extended version of Hugo installed
- [X] Used the exampleSite's config.toml as a reference

**Describe the feature**

I just set up a site with Compose, and immediately got a reaction from users asking "why are there two menus on top of eachother on the page on mobile? I didn't think about this myself while setting it up, but now that it's brought to my attention it is quite odd.

This request is for adding an option for, or updating the theme in general to merge the two menus into one menu with sections. I imagine that when you open the top menu (which would be the only one after implementing this change), it would first list the top bar navigation links, and then have a separator saying `Docs Menu` for the default setup, after which the Docs links would be listed.

In short the two menus are strange, and take up a lot of real estate at the top of the view on mobile.

Something like this from top to bottom is what I imagine, the `=` is to emphasize a section separator:

```
Search the docs
Tutorials
Example
=== Docs Menu ===
Themes' Docs
Compose Docs
Install theme
...
```

Image highlighting the two menus stacked on top of each other:

![Stacked Menus](https://i.imgur.com/aAm0i6t.png)
